test/e2e/theia_clickhouse_test.go

```diff
@@ -0,0 +1,358 @@
+                   egressNetworkPolicyRuleName,
+                   egressNetworkPolicyRuleAction,
+                   egressNetworkPolicyType,
+                   tcpState,
+                   flowType,
+                   sourcePodLabels,
+                   destinationPodLabels,
+                   throughput,
+                   reverseThroughput,
+                   throughputFromSourceNode,
+                   throughputFromDestinationNode,
+                   reverseThroughputFromSourceNode,
+                   reverseThroughputFromDestinationNode)
+                   VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?)`
+	recordPerCommit = 1000
+	insertInterval  = 1
+	threshold       = 25
+	MaxInt32        = 1<<31 - 1
+)
+
+var targetTable = map[string]string{
+	".inner.flows_node_view":   "16",
+	".inner.flows_pod_view":    "20",
+	".inner.flows_policy_view": "27",
+	"flows":                    "49",
+}
+
+func TestTheiaStatusCommand(t *testing.T) {
+	data, _, _, err := setupTestForFlowVisibility(t, false, false, false)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer func() {
+		teardownTest(t, data)
+		teardownFlowVisibility(t, data, false)
+	}()
+
+	clientset := data.clientset
+	kubeconfig, err := data.provider.GetKubeconfigPath()
+	require.NoError(t, err)
+	connect, pf, err := commands.SetupClickHouseConnection(clientset, kubeconfig, "", false)
+	require.NoError(t, err)
+	if pf != nil {
+		defer pf.Stop()
+	}
+
+	t.Run("testTheiaGetClickHouseDiskInfo", func(t *testing.T) {
+		testTheiaGetClickHouseDiskInfo(t, data)
+	})
+	t.Run("testTheiaGetClickHouseTableInfo", func(t *testing.T) {
+		testTheiaGetClickHouseTableInfo(t, data, connect)
+	})
+	t.Run("testTheiaGetClickHouseInsertRate", func(t *testing.T) {
+		testTheiaGetClickHouseInsertRate(t, data, connect)
+	})
+
+}
+
+func testTheiaGetClickHouseDiskInfo(t *testing.T, data *TestData) {
+	// retrieve metrics
+	stdout, err := getClickHouseDBInfo(t, data, getDiskInfoCmd)
+	require.NoError(t, err)
+	resultArray := strings.Split(stdout, "\n")
+	assert := assert.New(t)
+	length := len(resultArray)
+	assert.GreaterOrEqualf(length, 2, "stdout: %s", stdout)
+	// Check header component
+	assert.Containsf(stdout, "Shard", "stdout: %s", stdout)
+	assert.Containsf(stdout, "DatabaseName", "stdout: %s", stdout)
+	assert.Containsf(stdout, "Path", "stdout: %s", stdout)
+	assert.Containsf(stdout, "Free", "stdout: %s", stdout)
+	assert.Containsf(stdout, "Total", "stdout: %s", stdout)
+	assert.Containsf(stdout, "Used_Percentage", "stdout: %s", stdout)
+	for i := 1; i < length; i++ {
+		// check metrics' value
+		diskInfoArray := strings.Fields(resultArray[i])
+		assert.Equal(9, len(diskInfoArray), "number of columns is not correct")
```

Comment:
consider declare one or some variables or const for `9`, `default`, and `/var/lib/clickhouse/`.

Reply:
Updated. Thanks.